### DISCUSSION TITLE
why isn't the cryptroot plugin executable

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#188] [General]
like all the other plugins? 

```
...
-rwxr-xr-x 1 pi pi  2449 Jan 23 08:45 copydir
-rwxr-xr-x 1 pi pi  5920 Jan 23 08:45 copyfile
-rw-r--r-- 1 pi pi  7958 Feb 25 20:39 cryptroot
-rwxr-xr-x 1 pi pi  3871 Feb 25 20:39 disables
-rwxr-xr-x 1 pi pi  1910 Jan 23 08:45 explore
...
```

**Comment by gitbls** [comment]
Is this on a system built from an sdm customized IMG? If not, what is its genealogy?

**Comment by tinker2much** [comment]
Git cloned.On Feb 25, 2024, at 11:41 PM, Benn ***@***.***> wrote:﻿
Is this on a system built from an sdm customized IMG? If not, what is its genealogy?

**Comment by tinker2much** [comment]
Sorry, that was too terse. I forget how I created this system. I had git cloned Sdm to my Documents folder. I did a git pull yesterday to update it and noticed that one particular  file was 644 instead of 755 like the rest. On Feb 26, 2024, at 9:57 AM, Peter Lindgren ***@***.***> wrote:﻿Git cloned.On Feb 25, 2024, at 11:41 PM, Benn ***@***.***> wrote:﻿
Is this on a system built from an sdm customized IMG? If not, what is its genealogy?

**Reply by gitbls** [reply]
Ah! I couldn't see how sdm or EZsdmInstaller could do that, but indeed, wrong protection if you git clone. 

Corrected. Thx!
